styles/styles.css

```
@@ -94,49 +168,115 @@ body {
 /* end slider */
 
 /* start banner */
-
+.banner {
+  display: block;
+  margin-bottom: 81px;
+}
 .banner-header{
+  width: 434px;
+  margin: auto auto;
   font-weight: 900;
   font-size: 32px;
   line-height: 46px;
+  margin-bottom: 55px;
+
+}
+.banner-container {
+  display: flex;
+  justify-content: space-between;
 }
 
 .banner-list {
   background-color: #ff7799;
+  border-radius: 16px;
+  margin: 0;
+  padding: 0;
+}
+
+.banner-item {
+  display: flex;
+  flex-direction: column;
+  width: 570px;
 }
 
 .banner-title {
+  display: block;
+  margin-left: 36px;
+  margin-top: 36px;
+  margin-bottom: 28px;
   font-weight: 700;
   font-size: 24px;
   line-height: 30px;
 }
 
+.banner-image {
+  align-self: flex-end;
+}
 .banner-text {
+  display: block;
+  width: 304px;
+  margin: 0 ;
+  margin-bottom: 28px;
+  margin-left: 36px;
   line-height: 22px;
 }
 
 /* end banner */
 
 /* start assortment */
 
+.assortment {
+  margin-bottom: -11px;
+}
 .assortment-header {
+  display: block;
+  width: 527px;
+  margin: auto auto;
   font-weight: 900;
   font-size: 32px;
   line-height: 46px;
+  margin-bottom: 56px;
+}
+
+.assortment-list {
+display: flex;
+justify-content: space-between;
+flex-wrap: wrap;
+padding: 0;
+}
```

```suggestion
.assortment-list {
  display: flex;
  justify-content: space-between;
  flex-wrap: wrap;
  padding: 0;
}
```
Проверь везде пожалуйста, и лишние пустые строки убери